web/client/selectors/featuregrid.js

```diff
@@ -46,9 +46,13 @@ const hasGeometrySelectedFeature = (state) => {
 };
 const hasChangesSelector = state => changesSelector(state) && changesSelector(state).length > 0;
 const hasNewFeaturesSelector = state => newFeaturesSelector(state) && newFeaturesSelector(state).length > 0;
+const getAttributeFilters = state => state && state.featuregrid && state.featuregrid.filters;
```

Comment:
no use of get here?

Reply:
If it does not change, this version is more efficient